### ISSUE TITLE
any suggestions on how to speed up optimization?

### ISSUE THREAD

**Issue by imluchang** [neonine2/morpheus-spatial#8]
Setting `num_workers > 1` as suggested in the tutorial did speed optimization up, but Im looking for more speed up in generating the counterfactual explanations. I tried following the instructions for using Ray with slurm in the tutorial but I'm having trouble following. Are there plans to release tutorial for multi-node parallelization?

**Comment by neonine2** [comment]
Absolutely, stay tuned! 🚀 We are working on a tutorial right now that will use the `submitit` package to easily fan out jobs across hundreds of CPU nodes
